test/navigator/BorderNavigatorSpec.scala

```diff
@@ -60,24 +62,69 @@ class BorderNavigatorSpec extends SpecBase with ScalaCheckPropertyChecks with Ge
 
         }
 
-        //TODO: Change more information page to other page when created
-        "to more information page when security mode of transport at border is  5, security is 0 and active border transport is  present " in {
+        "to more information page when security mode of transport at border is not 5" +
+          "and security is 1,2,3" +
```

Comment:
As above. to make the tests more readable to someone not accustomed  to this service maybe use the config.Constants